src/components/card/Card.css

```diff
@@ -104,6 +104,6 @@
   .section-card {
     width: 500px;
     position: relative;
-    margin: 0 auto;
+    margin: 15px auto 15px auto;
```

Comment:
Should be the same as `15px auto` no?

Reply:
You are right! Changed the `margin` to `15px auto`. :)